src/mlagility/analysis/util.py

```diff
@@ -80,6 +82,53 @@ def count_parameters(model: torch.nn.Module, model_type: build.ModelType) -> int
     raise AnalysisException(f"model_type {model_type} is not supported")
 
 
+def get_onnx_ops_list(onnx_model) -> Dict:
+    """
+    List unique ops found in the onnx model
+    """
+    onnx_ops_counter = defaultdict(int)
+    try:
+        model = onnx.load(onnx_model)
+    except Exception as e: # pylint: disable=broad-except
+        print (f"Failed to get ONNX ops list from {onnx_model}: {str(e)}")
+        return "ONNX model analysis failed"
```

Comment:
The typing hint for `get_onnx_ops_list` indicates that a Dictionary is returned. However, we are returning a string here. Wound't it be more appropriate to return an empty dictionary here?